tests/rados/test_bluestore_configs.py

```diff
@@ -0,0 +1,111 @@
+""" Module to verify scenarios related to BlueStore config changes"""
+import time
+
+from ceph.ceph_admin import CephAdmin
+from ceph.rados.core_workflows import RadosOrchestrator
+from tests.rados.monitor_configurations import MonConfigMethods
+from tests.rados.stretch_cluster import wait_for_clean_pg_sets
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def run(ceph_cluster, **kw):
+    """
+    Common test module to verify BlueStore config changes
+    and functionalities.
+    Currently, covers the following tests:
+     - CEPH-83571646: Checksum Algorithms
+    """
+    log.info(run.__doc__)
+    config = kw["config"]
+    cephadm = CephAdmin(cluster=ceph_cluster, **config)
+    rados_obj = RadosOrchestrator(node=cephadm)
+    mon_obj = MonConfigMethods(rados_obj=rados_obj)
+
+    def restart_osd_service():
+        osd_services = rados_obj.list_orch_services(service_type="osd")
+        for osd_service in osd_services:
+            cephadm.shell(args=[f"ceph orch restart {osd_service}"])
+        time.sleep(30)
+
+    def create_pool_write_iops(param, pool_type):
+        pool_name = f"{pool_type}_pool_{param}"
+        assert (
+            rados_obj.create_pool(pool_name=pool_name)
+            if "repli" in pool_type
+            else rados_obj.create_erasure_pool(
+                name=pool_name, **{"pool_name": pool_name}
+            )
+        )
+
+        if param == checksum:
+            # set checksum value for the pool
+            cephadm.shell([f"ceph osd pool set {pool_name} csum_type {param}"])
+        # rados bench will perform IOPs and also verify the num of objs written
+        assert rados_obj.bench_write(pool_name=pool_name, **{"max_objs": 500})
```

Comment:
If pool level checksum verify command is available, can we raise an RFE for the same. Rest looks good to me.